packages/ia-topnav/src/data/petabox_signedout_menus.js

```diff
@@ -0,0 +1,509 @@
+/* eslint-disable */
+export default {
+  "audio": {
+    "heading": "Internet Archive Audio",
+    "iconLinks": [
+      {
+        "icon": "https://www-shaner4.archive.org/services/img/etree",
+        "title": "Live Music Archive",
+        "url": "https://www-shaner4.archive.org/details/etree"
```

Comment:
BLOCKER: All these links are using the dev host.

Reply:
1. This is only here for testing purposes to provide a way to test Petabox data without having to base64 encode it
2. The feature this PR completes is switching from always using the static menu data in the menus.js file in this directory to always requiring the menu data be provided to it on instantiation. This means that no matter what's in this test data, it won't make it to production because it's not being imported anywhere in the module files.